client/lib/domains/index.js

```diff
@@ -2,9 +2,12 @@
  * External dependencies
  */
 import inherits from 'inherits';
-import some from 'lodash/some';
-import includes from 'lodash/includes';
-import find from 'lodash/find';
+import {
+	endsWith,
+	find,
+	includes,
+	some
+} from 'lodash';
```

Comment:
We don't do direct imports from 'lodash' since it's gonna bring the whole `lodash` library here.